dist/angular-chart.js

```diff
@@ -86,16 +86,25 @@
       return {
```

Comment:
You shouldn't update the distribution files, this is done automatically when I publish a version.